darts/models/baselines.py

```diff
@@ -97,3 +99,26 @@ def predict(self, n: int):
         last_value = last + slope * n
         forecast = np.linspace(last, last_value, num=n + 1)[1:]
         return self._build_forecast_series(forecast)
+
+
+class NaiveEnsembleModel(EnsembleModel):
+
+    def __init__(self, models: List[ForecastingModel]):
+        """ Naive combination model
+
+        Naive implementation of `EnsembleModel`
+        Returns the average of all predictions of the constituent models
+        """
+        super().__init__(models)
+
+    def fit(self, training_series: TimeSeries, target_series: Optional[TimeSeries] = None) -> None:
+        super().fit(training_series, target_series)
+
+        for model in self.models:
+            if isinstance(model, UnivariateForecastingModel):
+                model.fit(training_series)
+            else:
+                model.fit(training_series, target_series)
```

Comment:
This isn't very clean, but I had to resort to this because calling `model.fit(training_series, target_series)` on a univariate forecasting model raises an error.

 This hints too towards removing the distinction between `UnivariateForecastingModel` and `MultivariateForecastingModel` as suggest in #231.

Reply:
Yes, don't worry too much about it, it will disappear if everything goes as planned :)